Build/libHttpClient.Linux/openssl_Linux.bash

```diff
@@ -42,6 +42,13 @@ if [ ! -d /usr/local/ssl/include/openssl ] ; then
     exit 1
 fi
 
+if [ -f "$SCRIPT_DIR/../../Out/x64/$CONFIGURATION/libcrypto.Linux/libcrypto.a" ]; then
```

Comment:
What about libssl or should we not worry about it since the expectation is that if crypto is built then so should ssl?

Reply:
The latter - since we're building OpenSSL via script and not source, the script will trigger the libcrypto build each time, prior to building libssl. I also wanted to match the behavior of the iOS build script.